packages/devextreme/js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -322,6 +327,12 @@ const DropDownEditor = TextBox.inherit({
         this.callBase();
     },
 
+    _setFieldAriaLabel() {
```

Comment:
I think that we can use ['_setFieldAria'](https://github.com/EugeniyKiyashko/DevExtreme/blob/9fa928b2af16c227295fac7459aa2365ef6b7d6e/js/ui/text_box/ui.text_editor.base.js#L488) method. 
Also I suggest test the following scenarios: 
- aria-label is declared using 'inputAttr' option
- change label, labelMode at runtime, `aria-labelledby` should be removed if label was deleted and vise versa